spec/lib/api/api_config_spec.rb

```diff
@@ -22,11 +22,19 @@
 
     describe 'identifiers' do
       let(:api_feature_identifiers) do
-        feature_identifiers { |set, id| set.add(id) }
-      end
+        collection_settings.each_with_object(Set.new) do |(_key, config), ids|
+          action_types = config.keys.select { |k| k.to_s.end_with?("_actions") }
+          action_identifiers = action_types.each do |action_type|
+            config[action_type].each do |_verb, actions|
+              actions.map do |action|
+                ids.merge(Array(action[:identifier]))
+                ids.merge(Array(action[:identifiers]).map { |i| i[:identifier] }.compact)
+              end
+            end
+          end
 
-      let(:sui_product_features) do
-        feature_identifiers { |set, id| set.add(id) if /^sui_/ =~ id }
```

Comment:
This is basically just taking a subset of the features from the full features list, but then later it checks:

```ruby
expect(sui_product_features.subset?(api_feature_identifiers))
```

which is _always_ going to be true.  This test makes no sense.

Reply:
WAT.  I don't understand either.

Reply:
It looks like you more than cover the rest of the changes since this one wasn't even really testing anything that either of us can tell...